README.md

```diff
@@ -66,13 +66,13 @@ npm install @google-cloud/dataproc
 'use strict';
 
 function main(projectId, region, clusterName, jobFilePath) {
-  const dataproc = require('@google-cloud/dataproc').v1;
+  const dataproc = require('@google-cloud/dataproc');
```

Comment:
curious about this one, is it generated by synthtool and something intentional?

Reply:
The change made for fixing client can't be found in samples/*.js
dataproc refer to dataproc.v1 and client is also looking for dataproc.v1.ClusterControllerClient
it break the path of finding client
(https://github.com/googleapis/nodejs-dataproc/blob/master/samples/createCluster.js)
```
const dataproc = require('@google-cloud/dataproc').v1;
...
 const client = new dataproc.v1.ClusterControllerClient(
...
);
```
Let me know if you have concern for this change.

Reply:
This change LGTM. We don't want to advertise `v1` (it's the default version anyway).

Reply:
...but I thought this README is auto-generated, is it produced by `synthtool` like this?

Reply:
> ...but I thought this README is auto-generated, is it produced by `synthtool` like this?

Yes. It is auto-generated by synthtool.